### PR TITLE
LIME-437: Add and delete workouts manually

### LINKED ISSUE #437: Add a form for adding workouts manually
1. Add a button and form for adding workouts manually. 

Image

### PR DESCRIPTION
**Pull request by stpalamar**
**The delete button is available only for workouts without a provider**
**km and km/h converts to meters and m/s**

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/26843161/3a1d4ca3-3329-4c36-a2de-5ec77a2e27ad

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/26843161/95b94835-307c-4df1-ac37-a5aa35cbfa21

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/26843161/09bdb131-8eea-4d88-bb92-5e2c37ad3771

### CODE REVIEW

File: frontend/src/bundles/workouts/components/create-workout-form/constants/constants.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    activityType: ActivityType.CYCLING,
+    workoutDate: configureDateString(new Date().toISOString()),
+    workoutStartedAt: configureTimeString(new Date().toISOString()),
+    workoutEndedAt: '',
+    speed: '',
+    distance: '',
+    steps: '',
+    heartRate: '',
+    kilocalories: '',
+    provider: null,
```
Comment (dimapopovych) [inline frontend/src/bundles/workouts/components/create-workout-form/constants/constants.ts]: Why do we need provider here?

File: frontend/src/bundles/workouts/components/create-workout-form/constants/constants.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+};
+
+const WORKOUT_ACTIVITY_OPTIONS = [
```
Comment (dimapopovych) [inline frontend/src/bundles/workouts/components/create-workout-form/constants/constants.ts]: I think we already have similar selector with ActivityType somewhere, maybe we could reuse options?

File: shared/src/helpers/configure-date/configure-date.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const configureDate = (
```
Comment (dimapopovych) [inline shared/src/helpers/configure-date/configure-date.ts]: We don't have similar helper? And as I can see you are not using it on backend, so maybe worth to move it to frontend and utilize date-fns to simplify

File: shared/src/helpers/configure-time-string/configure-time-string.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+function configureTimeString(isoString: string | null): string {
```
Comment (dimapopovych) [inline shared/src/helpers/configure-time-string/configure-time-string.ts]: same question for this one

**Review by dimapopovych — CHANGES_REQUESTED** [review]

**Review by dimapopovych — DISMISSED** [review]

**Review by SergiiTernytskyi — DISMISSED** [review]

**Comment by stpalamar** [comment]
Adapted for new sub-navigation

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/26843161/f2021d1e-fb65-4df2-9cec-e7a3fae3a75b

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**